forum/csakajs/gyak.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @typedef {Object} User
+ * @property {number} id
+ * @property {string} first_name
+ * @property {string} last_name
+ * @property {string} email
+ * @property {number} birthdate
+ */
+
+////////////////////read csv
+const fs = require("fs");
+const { parse } = require("path");
+fs.readFile("csv.csv", "utf8", (err, data) => {
+  if (err) {
+    console.error(err);
+    return;
+  }
+  getArray(data, 2);
+});
+////////////////////get array
+
+function getArray(data) {
+  const writeableData = data;
+  const result = [];
+  const lines = data.split("\n");
+  let headers = lines[0].split(";");
+
+  lines.map((l) => {
+    const obj = {};
+    const line = l.split(";");
+
+    headers.map((h, i) => {
+      obj[h] = line[i];
+    });
+
+    result.push(obj);
+  });
+  result.shift();
+  input(result, data);
+}
+////////////////////create readline
+const readline = require("readline");
```

Comment:
This doesn't "create" a readline, it just imports the readline package.
Please remove the comment
```suggestion
const readline = require("readline");
```